scss/base/_helpers.scss

```diff
@@ -0,0 +1,70 @@
+@mixin fix-inline-blocks($fontsize: 16px) {
+  font-size: 0;
+
+  > * {
+    display: inline-block;
+    font-size: $fontsize;
+  }
+}
+
+.cineva {
+  @include fix-inline-blocks(16px);
+}
+
+.section {
+  padding-top: 20px;
+
+  @include breakpoint($desktop) {
+    padding-top: 40px; // va aplica de la 1200 in sus
+  }
+}
+
+// exemplu desktop-first
+.section__title {
+  font-size: 20px;
+
+  @include breakpoint(max-width $tablet) {
+    font-size: 14px;  // va aplica doar de la 768 in jos
+  }
+}
+
+// exemplu de range 
+.section__content {
+  margin-bottom: 20px;
+
+  @include breakpoint($tablet $desktop) {
+    margin-bottom: 40px; // va aplica doar intre 768 si pana la 1200
+  }
+}
+
+.ratio-box1 {
```

Comment:
you now have a duplicate for these styles - you have a `ratio-box` here and then the same code repeated in `components/_ratio-box.scss`. Please pick one of them and delete the other. If you decide to keep this - please name the class using just `ratio-box`, no need to add the `1` here